Makefile.custom.mk

```diff
@@ -0,0 +1,34 @@
+##@ Custom
+
+HELM_SCHEMA_VERSION := 0.11.1
+HELM_SCHEMA := ./bin/helm-schema-$(HELM_SCHEMA_VERSION)
+
+HELM_UNITTEST_VERSION := 0.5.0
+HELM_UNITTEST := ./bin/helm-unittest-$(HELM_UNITTEST_VERSION)
+
+.PHONY: gen-schma
+gen-schema: $(HELM_SCHEMA) ## Generates schema.
+	$(HELM_SCHEMA) -l debug
```

Comment:
I would be careful here. `helm-schema` sets `additionalProperties: false` by default thus blocking possible additional properties. I see you took care of this in your manual schema annotations. And it seems to work how you intended. Just wanted to mention this.

Reply:
I tested it on GS WC and it worked fine.